rest/models/DashboardTemplate.go

```diff
@@ -16,7 +16,7 @@ import (
 type AvailableTemplates string
 
 const (
-	LandingPage AvailableTemplates       = "landingPage"
+	LandingPage       AvailableTemplates = "landingPage"
```

Comment:
@JetyAdam I checked locally and my IDE (vscode) also modifies these whitespaces - but only when I modify the file. Since we don't have any logic changes in this file, can we just reset this file with upstream to avoid merge conflicts?